packages/scenario/test/relevant.test.ts

```diff
@@ -572,4 +573,68 @@ describe('Relevance - TriggerableDagTest.java', () => {
 			});
 		}
 	);
+
+	/**
+	 * **PORTING NOTES**
+	 *
+	 * Deals with intersection of repeats and relevance. Organizing tests like
+	 * these is always going to be subjective, but I'll throw it out there now
+	 * that it might make sense to have more specific suites/modules
+	 * ("bags"/"vats" lol) rather than trying to lump them into one concern or the
+	 * other. If for no other reason than intersecting functionality being
+	 * particularly thorny in terms of essential complexity, which could make for
+	 * a good organizational principle for these sorts of tests.
+	 */
+	describe('when repeat and top-level node have [the] same relevance expression, and [the] expression evaluates to false', () => {
+		/**
+		 * **PORTING NOTES**
+		 *
+		 * Rephrase?
```

Comment:
In hindsight, I think it might have been better to methodically:

1. Copy the full JavaRosa test method name verbatim
2. Phrase as seems appropriate for the format
3. Align however/whenever we see fit case-by-case

Given that's hindsight, maybe that's something we could do piecemeal as specific tests come into focus? At least with the current approach, I should still be able to trace each test back to the JR source without too much effort.

Reply:
Sounds good 👍